.travis/test.ls

```diff
@@ -0,0 +1,30 @@
+require! \stream-snitch
+pty = require \pty.js
+
+urbit =
+  # TODO abort on failure
+  pty.spawn './urbit' <[-B urbit.pill -A .. -cFI zod zod]> 
+     .on \data -> process.stdout.write it
+     
+urbit.on \exit (code)->
+  console.log "\nnode: urbit exited with code #code\n"
+  process.exit code
+
+fin = no
+urbit.pipe (new stream-snitch /dojo> /g).on \match ->
+    return if fin
+    fin := yes
+    console.log "\n\n---\nnode: got dojo!\n---\n\n"
+    set-timeout (-> process.exit 0), 1000 # should probably test further
+    
+urbit.pipe (new stream-snitch /\n(\/~|ford: )/g).on \match ->
+    return if fin
+    fin := yes
+    console.log "\n\n---\nnode: detected error\n---\n\n"
+    set-timeout (-> process.exit 1), 1000
+    
+set-timeout ...
```

Comment:
This should be removed to let travis handle the timeout. If we do leave it in here, we should have it exit.